api/MediaDevices.json

```diff
@@ -503,6 +503,546 @@
             "standard_track": true,
             "deprecated": false
           }
+        },
+        "constraints_aspectRatio_parameter": {
+          "__compat": {
+            "description": "`constraints.aspectRatio` parameter",
```

Comment:
How about instead:

(Note: The `returns_` prefix is [quite common](https://github.com/search?q=repo%3Amdn%2Fbrowser-compat-data+%2Freturns_%2F+path%3A**.json&type=code).)

```
        "returns_constraints_property": {
          "aspectRatio": {
            "__compat": {
              "description": "`constraints.aspectRatio` in returned object",
```

Reply:
Arrgg. It has to be "returns". I'm going to fix up.